freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/stack/model/common/instance/azure/AzureInstanceTemplateEncryptionParameters.java

```diff
@@ -0,0 +1,40 @@
+package com.sequenceiq.freeipa.api.v1.freeipa.stack.model.common.instance.azure;
+
+
+import com.sequenceiq.common.api.type.EncryptionType;
+import com.sequenceiq.freeipa.api.v1.freeipa.stack.doc.FreeIpaModelDescriptions;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel(value = "AzureInstanceTemplateEncryptionParameters")
+public class AzureInstanceTemplateEncryptionParameters {
```

Comment:
Like for `AzureInstanceTemplateParameters`, I do not feel we need such a degree of customization here, so I would just drop this class as well.